assemblies/assembly-dashbuilder.adoc

```diff
@@ -0,0 +1,46 @@
+[id='assembly-dashbuilder_{context}']
+= Installing and Using Dashbuilder and Dashbuilder runtimes
+ifdef::context[:parent-context: {context}]
+
+:context: dashbuilder
+
+== Installing on EAP
+// assembly-install-on-eap
+include::{enterprise-dir}/admin-and-config/dashbuilder-runtimes-con.adoc[leveloffset=+2]
+include::{enterprise-dir}/admin-and-config/installing-dashbuilder-proc.adoc[leveloffset=+2]
+include::{enterprise-dir}/admin-and-config/datasets-dashbuilder-runtimes-con.adoc[leveloffset=+2]
+include::{enterprise-dir}/admin-and-config/customizing-dashbuilder-con.adoc[leveloffset=+2]
+
+== Configuring {CENTRAL}
+//[id='assembly-configuring-central']
+
+//Dashbuilder Data Transfer*: Import and export Dashbuilder data as ZIP files in {CENTRAL}.
+include::{jbpm-dir}/BAM/Dashbuilder-data-management-con.adoc[leveloffset=+2]
+include::{jbpm-dir}/BAM/Exporting-dashbuilder-data-proc.adoc[leveloffset=+2]
+include::{jbpm-dir}/BAM/Importing-dashbuilder-data-proc.adoc[leveloffset=+2]
+include::{jbpm-dir}/BAM/Deploying-dashbuilder-data-proc.adoc[leveloffset=+2]
+
+== Building Custom Dashboard Widgets
+//assembly-building-custom-dashboard-widgets
+include::{enterprise-dir}/pages/building-custom-dashboard-widgets-heatmap-components-con.adoc[leveloffset=+2]
+include::{enterprise-dir}/pages/building-custom-dashboard-widgets-creating-process-heatmap-components-proc.adoc[leveloffset=+3]
+include::{enterprise-dir}/pages/building-custom-dashboard-widgets-creating-all-processes-heatmap-components-proc.adoc[leveloffset=+3]
+include::{enterprise-dir}/pages/building-custom-dashboard-widgets-executing-heatmap-component-proc.adoc[leveloffset=+3]
+]
+//It looks like heat maps apply to Dashbuilder. Is that right?
```

Comment:
@emmurphy1 I believe it applies to Dashbuilder. @mramendi Is this correct and does it need to be retained in assembly-building-custom-dashboard-widgets.adoc ?